contracts/LiquidityBridgeContract.sol

```diff
@@ -409,11 +517,17 @@ contract LiquidityBridgeContract {
             s := mload(add(signature, 0x40))
             v := byte(0, mload(add(signature, 0x60)))
         }
+		// SDL: I'm think we should not use this signature format
+		// but an EIP712 compatible format.
```

Comment:
This is compatible with web3. I would have to read EIP712 and see if it can be easily implemented in Go/web3. I suggest to leave this as future work.

Reply:
ok. But note that most web3 providers also support EIP712 signatures.